tests/unittests.py

```diff
@@ -51,6 +51,16 @@ def test_limits_option(self):
                                    limits={'cpu': 2, 'memory': '10m'})
         self.assertEqual(service.limits, {'cpu': 2, 'memory': '10m'})
 
+    def test_no_ports_option(self):
+        service = entities.Service('foo', 'stackbrew/ubuntu:13.10')
+        self.assertEqual(service.ports, {})
+
+    def test_ports_option(self):
+        service = entities.Service('foo',
+                                   'stackbrew/ubuntu:13.10',
+                                   ports={'server: 4848'})
+        self.assertEqual(service.ports, {'server: 4848'})
+
```

Comment:
Add a test for containers to check that ports defined at the service level correctly propagate down to the container instances' ports.